File: app/src/main/java/com/sixbeeps/uniquity/data/UnicodeCharacter.java
package com.sixbeeps.uniquity.data;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.util.List;

@Entity
public class UnicodeCharacter {
    @PrimaryKey
    @NonNull
    public String character = "";
    public String byteRepresentation;
    public String group;
}

File: app/src/main/java/com/sixbeeps/uniquity/data/UnicodeCharacter.kt
package com.sixbeeps.uniquity.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class UnicodeCharacter {
    @JvmField
    @PrimaryKey
    var codepoint: String = ""

    @JvmField
    var name: String? = null

    @JvmField
    var groupName: String = ""
}

